classes/submission/reviewRound/ReviewRound.php

```diff
@@ -151,6 +154,11 @@ public function setStatus($status)
      */
     public function determineStatus()
     {
+        // The submission back form copy editing stage to last review round
+        if ($this->getStatus() == self::REVIEW_ROUND_STATUS_RETURNED_TO_REVIEW) {
+            return self::REVIEW_ROUND_STATUS_RETURNED_TO_REVIEW;
+        }
+
```

Comment:
what can we do is to move this block in some later part of `determineStatus` function so that before showing status such as `Retune back to review`, it can calculate/determine other more important status such as `Resubmit for Review` or other more important action that require attention right before it passed to copy editing stage .

Reply:
Good idea. We can put it just before `REVIEW_ROUND_STATUS_REVIEWS_COMPLETED` so, at least, the round status based on review assignments will be shown.

Reply:
Done .